model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -1964,13 +1964,13 @@ message ExecutableStagePayload {
 message StandardResourceHints {
   enum Enum {
     // Describes hardware accelerators that are desired to have in the execution environment.
-    // Payload: ASCII encoded string with the following format: "type:<type>;count:<n>;<options>" where type
-    // is an accelerator sku, count is the number of accelerators per worker, and options are
+    // Payload: beam:coder:string_utf8:v1 encoded string with the following format: "type:<type>;count:<n>;<options>" where type
+    // is an accelerator SKU, count is the number of accelerators per worker in the range [0, 2^64), and options are
     // related options flags.
     ACCELERATOR = 0 [(beam_urn) = "beam:resources:accelerator:v1"];
     // Describes desired minimal available RAM size in transform's execution environment.
     // SDKs should convert the size to bytes, but can allow users to specify human-friendly units (e.g. GiB).
-    // Payload: ASCII encoded string of the base 10 representation of an integer number of bytes.
+    // Payload: beam:coder:string_utf8:v1 encoded string of the number of bytes in the range [0, 2^64).
```

Comment:
I think it's still worth clarifying it's in base 10, unless we are also accepting Hex or base 64 or other encodings. I'd rather be strict than allow unnecessary wiggle room. It can always be expanded later.